Governance: Create token governance (#1968)
solana-labs/solana-program-library: governance/README.md

taken under governance signed the transaction. Optionally `CreateProgramGovernance` instruction can also transfer `upgrade_authority`
of the governed program to the Governance PDA at the creation time of the Governance account.

### Mint Governance account

> Mint governance account allows to setup a governance over a Mint account.

Reviewer: nit
```suggestion
A mint governance account allows a mint authority to setup governance over an SPL Mint account.
```

> The Governance program validates at creation time the current mint authority sign the transaction to

Reviewer: nit:
```suggestion
The Governance program validates at creation time that the current mint authority signed the transaction to
```

create the governance and optionally can transfer the authority to the Governance account.

> Once setup the Mint Governance allows to create Proposals to execute mint instructions for the governed Mint.

Reviewer: nit:
```suggestion
Once setup the Mint Governance allows governance participants to create Proposals which execute mint instructions for the governed Mint.
```

### Token Governance account

> Token governance account allows to setup a governance over a Token account.

Reviewer: ```suggestion
A token governance account allows a token account owner to setup governance over an SPL Token account.
```

> The Governance program validates at creation time the current owner sign the transaction to

Reviewer: nit ```suggestion
The Governance program validates at creation time that the current owner signed the transaction to
```

Author: Done

create the governance and optionally can transfer the owner to the Governance account.

> Once setup the Token Governance allows to create Proposals to execute transfer instructions from the governed token account.

Reviewer: ```suggestion
Once setup the Token Governance allows participants to create Proposals to execute transfer instructions from the governed token account.
```

by the Solana program deploy cli command.
However, in order for Governance to be useful, Governance now needs this authority.

In similar fashion for Mint and Token governances the relevant authorities to mint and transfer tokens

> are transferer to the Governance account. It in turns allows to create and vote on Proposals which can then execute

Reviewer: ```suggestion
are transferred to the Governance account. It in turn allows participants to create and vote on Proposals which can then execute
```
